File: JavaProject/src/com/enum1/EnumExample.java
package com.enum1;


	enum Day {//ENUMERATION DATA TYPE
	    MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY
	}//we fixed set of constant

	public class EnumExample {
	    public static void main(String[] args) {
	        Day today = Day.FRIDAY;
	        System.out.println("Today is: " + today);
	    }
	}


//They are particularly useful when you have a variable that can only take one out of a small set of possible values. 

File: PythonProject/enum/EnumExample.py
from enum import Enum

# Define the Day enum
class Day(Enum):
    MONDAY = 1
    TUESDAY = 2
    WEDNESDAY = 3
    THURSDAY = 4
    FRIDAY = 5
    SATURDAY = 6
    SUNDAY = 7

# Main logic
if __name__ == "__main__":
    today = Day.FRIDAY
    print("Today is:", today.name)
